changelog/unreleased/pull-2733

```diff
@@ -0,0 +1,6 @@
+S3 backend: Add support for WebIdentityTokenFile
```

Comment:
The build fails because of this line. It must start with Bugfix:, Change:, Enhancement: or Security:. In this case, I guess it should read:

    Enhancement: Add support for WebIdentityTokenFile in S3 backend

Reply:
I have changed this as advised